ad-bank.md

```diff
@@ -0,0 +1,39 @@
+PostHog Ad Bank
+<hr/>
+
+## POSTHOG SELF-HOSTED
+
+* PostHog: product analytics software that’s fast, powerful, and easy to use - all completely self-hosted for maximum SOC 2 & HIPAA compliance. See it in action at posthog.com/product
+* Looking to self-host your own open-source product analytics platform? Look no further than PostHog - a fully-featured platform with no 3rd party cookies. Learn more at posthog.com
+* With PostHog’s self-hosted product analytics platform, your data stays on your infrastructure and you get full access to your production instance. See a demo today at posthog.com 
+* Self-hosted product analytics *and* complete privacy? We’ve got you covered. PostHog’s self-hosted app analytics platform is the perfect solution for privacy-conscious companies like yours. Deploy it free today at posthog.com/signup
+* PostHog’s open-source product analytics can be self-hosted and is great for small teams than want to understand usage in their apps. Get it free today at posthog.com/signup 
+* Looking for a free, open-source product analytics solution that you can self-host? Try PostHog today - and retain complete control over your data. Deploy it free at posthog.com/signup
+* With PostHog’s self-hosted product analytics solution, your data remains on your servers - ensuring total privacy with no data leaks. Try it out today at posthog.com/product
+* PostHog’s self-hosted product analytics features a Postgres or Clickhouse database for your user data. Manage your server or let us manage it for you hassle-free. Learn more at posthog.com/signup 
+* Our self-hosted product analytics solution features unlimited event ingestions, unlimited user tracking, unlimited data retention, and multiple projects across unlimited teams. Sign up free at posthog.com/signup
+* PostHog: powerful product analytics with SSO, API access, user permissions, and ad blocker resistance. See how it works (and deploy it free) at posthog.com/product 
```

Comment:
I'm not sure if this features are ones we'd want to highlight. Permissioning, for example, is really rudimentary right now. 

There's some guidance here on what I think our clearest USPs and benefits are: https://posthog.com/handbook/growth/marketing/messaging_framework

Reply:
Amended